pkg/azuredx/models/macro.go

```diff
@@ -66,6 +66,13 @@ func (md MacroData) Interpolate(query string) (string, error) {
 	return interpolated, nil
 }
 
+func quoteForSpacesDotsDashes(s string) string {
+	if strings.ContainsAny(s, " .-") {
```

Comment:
can you add a link to the docs you found here as a comment? (for future readers)